app/models/request_information_type.rb

```diff
@@ -1,6 +1,41 @@
+# frozen_string_literal: true
+
+# A request information type associates a pipline with request information,
+# either via {Request::Metadata} or a {LabEvent}
 class RequestInformationType < ApplicationRecord
```

Comment:
All the logic here used to exist on Request, but I've re-written it. The formatters stuff isn't actually used, but seemed like a solid enough idea that I'v kept it for now.